java/src/jmri/jmrix/xpa/XpaTurnout.java

```diff
@@ -36,62 +29,47 @@
     public XpaTurnout(int number, XpaSystemConnectionMemo m) {
         super(m.getSystemPrefix() + "T" + number);
         _number = number;
+        _prefix = m.getSystemPrefix();
```

Comment:
Use .equals here.  Prefix is a string.

Reply:
I could replace ``=`` to compare  strings, but here I set  the value of ``_prefix``

Reply:
Your comment is attached to line 32. I have since removed ``_prefix`` variable form ``XpaTurnout`` following your next suggestion